homeassistant/components/climate/__init__.py

```diff
@@ -43,6 +43,9 @@
 STATE_AUTO = "auto"
 STATE_DRY = "dry"
 STATE_FAN_ONLY = "fan_only"
+STATE_AWAY_HOLD = "away"
+STATE_HOME_HOLD = "home"
+STATE_TEMPERATURE_HOLD = "temp"
```

Comment:
I like having them as constants, but I don't think they should be named states as that is something specific in home assistant, and these do not qualify. We should think of something else.

The hold mode service has now been implemented in the climate component, so it does make sense to keep these constants (renamed) here in the component. Although I'm not sure that the hold mode service is a universal name for what it does.

Maybe I'm being picky about all these names, but I think that having good and accurate names help reading and explaining the code and understanding how home assistant works.